README.md

```diff
@@ -1,19 +1,60 @@
-# Hybrid Computing Jupyter Notebooks
+# Hybrid Computing
 
-These notebooks demonstrate how you can apply hybrid solvers to your problem, 
-create hybrid workflows, and develop custom hybrid components. 
+These notebooks demonstrate how you can apply hybrid solvers to your problem,
+create hybrid workflows, and develop custom hybrid components.
 
-Notebook 01 will 
-start you off with Leap's cloud-based hybrid solvers and out-of-the-box 
-dwave-hybrid samplers and workflows, and requires only familiarity with the binary 
-quadratic model (BQM) approach to formulating problems for solution on quantum 
-computers. 
+* **Notebook 01** will start you off with Leap's cloud-based hybrid solvers and
+  out-of-the-box dwave-hybrid samplers and workflows, and requires only familiarity
+  with the binary quadratic model (BQM) approach to formulating problems for
+  solution on quantum computers.
+* **Notebook 02** shows how you create your own workflows using existing dwave-hybrid
+  components.
+* **Notebook 03** is aimed at developers interested in developing their own hybrid
+  components for optimal performance.
 
-Notebook 02 shows how you create your own workflows using existing dwave-hybrid 
-components. 
+## A Few Words on Hybrid Computing
 
-Notebook 03 is aimed at developers interested in developing their own hybrid components 
-for optimal performance.
+Quantum-classical hybrid is the use of both classical and quantum resources to
+solve problems, exploiting the complementary strengths that each provides. As
+quantum processors grow in size, offloading hard optimization problems to quantum
+computers promises performance benefits similar to CPUs' outsourcing of
+compute-intensive graphics-display processing to GPUs.
+
+This [Medium Article](https://medium.com/d-wave/three-truths-and-the-advent-of-hybrid-quantum-computing-1941ba46ff8c)
+provides an overview of, and motivation for, hybrid computing.
+
+D-Wave's [Leap quantum cloud service](https://cloud.dwavesys.com/leap) provides
+cloud-based hybrid solvers you can submit arbitrary BQMs to. These solvers, which
```

Comment:
Not just BQMs anymore; but not sure if we want to mention that here since dwave-hybrid hasn't been quite updated for DQM/CQM.

Reply:
changed to "quadratic models"